ui-v2/app/controllers/dc/kv/edit.js

```diff
@@ -10,7 +10,7 @@ export default Controller.extend({
   encoder: service('btoa'),
   setProperties: function(model) {
     // TODO: Potentially save whether json has been clicked to the model
-    set(this, 'json', false);
+    set(this, 'json', true);
```

Comment:
Good spot, the `json: false` should have been changed to `json: true` also.

But you bringing this up prompted me to think about why I was `set`ting it here also. So after checking this I must have previously decided that this setting should always reset itself to the default (previously off, but this PR is about changing it to true) every single time you visited the KV page. In order to do this I reset it here to the default so it resets on every visit to the KV form.

I'm still so in two minds over this, so just checked in with @opihana who's tipped me the other way. So we've changed this so the KV code toggle acts more like a global setting, instead of just for whilst you are in the edit form. I definitely think this is better, made a commit reflecting this.

I think further work will be putting this now global setting into localStorage to last between sessions.